docs/NSR.md

```diff
@@ -155,24 +135,9 @@ Each factor is independent of the other(s).
    2. store a Root CA Private Key; or
    3. create digital signatures using a Root CA Private Key.
 
-**SANS Top 25**: A list created with input from the SysAdmin, Audit, Network, and Security (SANS) Institute and the Common Weakness Enumeration (CWE) that identifies the Top 25 Most Dangerous Software Errors that lead to exploitable vulnerabilities (see <http://www.sans.org/top25-software-errors/>).
-
-**Security Support System**: A system or set of systems supporting the security of the CA Infrastructure, which minimally includes:
-
-   1. authentication;
-   2. network boundary control;
-   3. audit logging;
-   4. audit log reduction and analysis;
-   5. vulnerability scanning;
-   6. physical intrusion detection;
-   7. host-based intrusion detection; and
-   8. network-based intrusion detection.
-
 **System**: One or more pieces of equipment or software that stores, transforms, or communicates data.
 
-**Trusted Role**: An employee or contractor of a CA or Delegated Third Party who has authorized access to any component of CA Infrastructure.
-
-**Vulnerability Scan**: A process that uses manual or automated tools to probe internal and external systems to check and report on the status of operating systems, services, and devices exposed to the network and the presence of vulnerabilities listed in the NVD, OWASP Top Ten, or SANS Top 25.
+**Trusted Role**: An individual employee or contractor of a CA or Delegated Third Party who has authorized access to any component of CA Infrastructure.
```

Comment:
What's the idea behind the "individual" here? At least for me as a non-native speaker "an employee" and "an individual employee" sound identical...

Reply:
I added this in to address confusion expressed around whether an entire company, used by a CA, could be considered a Trusted Role.